### ISSUE TITLE
Mentorship Success - Adidas (Larry Hall & Mireille Saeed)

### ISSUE THREAD

**Issue by mikepromazo** [promazo/Content-Team#191]

**Comment by mikepromazo** [comment]
Raw videos: https://drive.google.com/drive/u/6/folders/1oVRZFvUldI0oszWB7VUMxAQElZTSHMHB

Transcription: https://www.rev.com/transcript-editor/shared/6NIXpdYixAtDuA8xgElLMJa_hKaCimVhKkk9LAuocY-Tmf5K3hUSDp_9LDGILBnyVli6KI1mF1ODJGbzzXdprQSafcI?loadFrom=SharedLink

**Comment by mikepromazo** [comment]
Medium Length LinkedIn Video Notes:

https://docs.google.com/document/d/1KkmJMPQuUDp6_cHzqZ5buzX-bjEJsvjDpWXBwdvEUmI/edit

**Comment by bryanace53** [comment]
Hey Mike,

Here is the rough draft cuts of the video no designs like last time. Once the rough cut is done I can add the design. 

https://drive.google.com/drive/u/4/folders/1ZFSIHliM7xLLpQYOvf0njrrHhy-gX61h

**Comment by mikepromazo** [comment]
Thanks Bryan I updated the notes and called out sections that should be cut from the medium version.  Let me know once you've made the cuts so I can review again and finalize the sections we want to include.

https://docs.google.com/document/d/1KkmJMPQuUDp6_cHzqZ5buzX-bjEJsvjDpWXBwdvEUmI/edit#

**Comment by bryanace53** [comment]
Hi here is the updated video

https://drive.google.com/drive/u/4/folders/1ZFSIHliM7xLLpQYOvf0njrrHhy-gX61h

**Comment by mikepromazo** [comment]
Edits: https://docs.google.com/document/d/1KkmJMPQuUDp6_cHzqZ5buzX-bjEJsvjDpWXBwdvEUmI/edit#heading=h.uwwaktlj5vqk

**Comment by bryanace53** [comment]
Hi here is the video I still have to fix the intro but wanted to send this out. I will double check again today on the flow of the video. https://drive.google.com/drive/u/4/folders/1ZFSIHliM7xLLpQYOvf0njrrHhy-gX61h

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1ZFSIHliM7xLLpQYOvf0njrrHhy-gX61h

Here is the updated video

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1ZFSIHliM7xLLpQYOvf0njrrHhy-gX61h

Updated Video
